api/source/basics.rst

```diff
@@ -99,232 +99,419 @@ Languages
-+---------------------------------+--------------------+
-| Russian                         | ru                 |
-+---------------------------------+--------------------+
-| Serbian                         | sr                 |
-+---------------------------------+--------------------+
-| Sinhala (Sinhalese)             | si                 |
-+---------------------------------+--------------------+
-| Slovak                          | sk                 |
-+---------------------------------+--------------------+
-| Slovenian                       | sl                 |
-+---------------------------------+--------------------+
-| Spanish                         | es                 |
-+---------------------------------+--------------------+
-| Spanish (Spain)                 | es-es              |
-+---------------------------------+--------------------+
-| Sundanese                       | su                 |
-+---------------------------------+--------------------+
-| Swahili                         | sw                 |
-+---------------------------------+--------------------+
-| Swedish                         | se                 |
-+---------------------------------+--------------------+
-| Tagalog (Filipino)              | tl                 |
-+---------------------------------+--------------------+
-| Tamil                           | ta                 |
-+---------------------------------+--------------------+
-| Telugu                          | te                 |
-+---------------------------------+--------------------+
-| Thai                            | th                 |
-+---------------------------------+--------------------+
-| Turkish                         | tr                 |
-+---------------------------------+--------------------+
-| Ukrainian                       | uk                 |
-+---------------------------------+--------------------+
-| Urdu                            | ur                 |
-+---------------------------------+--------------------+
-| Uzbek                           | uz                 |
-+---------------------------------+--------------------+
-| Vietnamese                      | vn                 |
-+---------------------------------+--------------------+
-| Xhosa                           | xh                 |
-+---------------------------------+--------------------+
-| Zulu                            | zu                 |
-+---------------------------------+--------------------+
+Transcription and translation are supported in the following languages for HIGH (PROFESSIONAL), MECHANICAL, IMPORT and STANDARD fidelity jobs (volume may vary contact `support <mailto:support@cielo24.com>`_ for more information):
+
+
+High
+**********************
+
++------------------------------------------+---------------------+---------------------+---------------------+
+| Code                                     | Language            | Supported as source | Supported as target |
```

Comment:
@BrunoMeyer , use the same table header in the old docs: https://cielo24.readthedocs.io/en/latest/basics.html#getting-started:~:text=for%20more%20information)%3A-,Language,IETF%20Language%20Code,-Arabic